Source/ObjectiveDropboxOfficial/Platform/ObjectiveDropboxOfficial_iOS/DBOAuthMobile-iOS.m

```diff
@@ -15,7 +15,7 @@
 
 @implementation DBMobileSharedApplication {
   UIApplication *_sharedApplication;
-  UIViewController *_controller;
+    __weak UIViewController * _Nullable _controller;
```

Comment:
nit: this file has 2-whitespace indent, maybe shorten this indent to align with other lines?